nautobot/core/templates/utilities/theme_preview.html

```diff
@@ -1135,4 +1151,88 @@ <h1>Navbars</h1>
+            <div class="icon-preview"><img alt="pin-3 icon" class="nb-icon" src="{% static 'icon/pin-3.svg' %}">pin-3</div>
+            <div class="icon-preview"><img alt="plug icon" class="nb-icon" src="{% static 'icon/plug.svg' %}">plug</div>
+            <div class="icon-preview"><img alt="rocket-2 icon" class="nb-icon" src="{% static 'icon/rocket-2.svg' %}">rocket-2</div>
+            <div class="icon-preview"><img alt="rotate-cw icon" class="nb-icon" src="{% static 'icon/rotate-cw.svg' %}">rotate-cw</div>
+            <div class="icon-preview"><img alt="route icon" class="nb-icon" src="{% static 'icon/route.svg' %}">route</div>
+            <div class="icon-preview"><img alt="secrets icon" class="nb-icon" src="{% static 'icon/secrets.svg' %}">secrets</div>
+            <div class="icon-preview"><img alt="server icon" class="nb-icon" src="{% static 'icon/server.svg' %}">server</div>
+            <div class="icon-preview"><img alt="server-2 icon" class="nb-icon" src="{% static 'icon/server-2.svg' %}">server-2</div>
+            <div class="icon-preview"><img alt="share icon" class="nb-icon" src="{% static 'icon/share.svg' %}">share</div>
+            <div class="icon-preview"><img alt="shield-check icon" class="nb-icon" src="{% static 'icon/shield-check.svg' %}">shield-check</div>
+            <div class="icon-preview"><img alt="sitemap-outline icon" class="nb-icon" src="{% static 'icon/sitemap-outline.svg' %}">sitemap-outline</div>
+            <div class="icon-preview"><img alt="star icon" class="nb-icon" src="{% static 'icon/star.svg' %}">star</div>
+            <div class="icon-preview"><img alt="star-filled icon" class="nb-icon" src="{% static 'icon/star-filled.svg' %}">star-filled</div>
+            <div class="icon-preview"><img alt="transform icon" class="nb-icon" src="{% static 'icon/transform.svg' %}">transform</div>
+            <div class="icon-preview"><img alt="wifi icon" class="nb-icon" src="{% static 'icon/wifi.svg' %}">wifi</div>
+        </div>
+    </div>
+    <div class="col-12">
+        <h2>Customizing size</h2>
+            <div class="column-gap-8 d-flex flex-wrap px-20 py-16 rounded row-gap-16" style="background-color: #121a25;">
+            <div class="icon-preview"><img alt="wifi icon" class="nb-icon-xs" src="{% static 'icon/wifi.svg' %}">nb-icon-xs</div>
+            <div class="icon-preview"><img alt="wifi icon" class="nb-icon-sm" src="{% static 'icon/wifi.svg' %}">nb-icon-sm</div>
+            <div class="icon-preview"><img alt="wifi icon" class="nb-icon" src="{% static 'icon/wifi.svg' %}">nb-icon,<br>nb-icon-md</div>
+            <div class="icon-preview"><img alt="wifi icon" class="nb-icon-lg" src="{% static 'icon/wifi.svg' %}">nb-icon-lg</div>
+            <div class="icon-preview"><img alt="wifi icon" class="nb-icon-xl" src="{% static 'icon/wifi.svg' %}">nb-icon-xl</div>
+        </div>
+    </div>
```

Comment:
Do we have a use case in mind for the icon-size feature at this time? If not let's remove it for now.